src/module/actors/character.ts

```diff
@@ -9,8 +9,8 @@ import {
 import { SWNRBaseItem } from "../base-item";
 
 export class SWNRCharacterActor extends Actor<SWNRCharacterData> {
-  getRollData(): Record<string, any> {
-    const data = super.getRollData() as Record<string, any>;
+  getRollData(): Record<string, unknown> {
+    const data = super.getRollData() as Record<string, unknown>;
```

Comment:
Once again, random clean ups are for a dedicated PR.

Reply:
Done as part of #32.